src/sinks/gcp/stackdriver_metrics.rs

```diff
@@ -21,11 +20,7 @@ pub struct StackdriverConfig {
     pub credentials_path: Option<String>,
     #[serde(default = "default_metric_namespace_value")]
     pub default_namespace: String,
-    #[serde(default)]
-    pub request: TowerRequestConfig,
-    #[serde(default)]
-    pub batch: BatchConfig,
-    pub tls: Option<TlsOptions>,
```

Comment:
At minimum, the `request` and `tls` options will need to be reintroduced, as they will become part of the component specification. Ideally, `max_batch_size` should use the `batch` options above as well for consistency with other components.